pkgs/servers/web-apps/wordpress/generic.nix

```diff
@@ -12,6 +12,15 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
+    # remove non-essential plugins and themes
+    rm -r wp-content/{plugins,themes}
+    mkdir wp-content/plugins
+    cat << EOF > wp-content/plugins/index.php
+    <?php
+    // Silence is golden.
```

Comment:
is that a safety net to prevent directory listing?

Reply:
Yeah probably. Also is in the official package. Maybe also add it to the themes folder? (Or am I missing something?)

Reply:
I guess yes, see upstream file https://github.com/WordPress/WordPress/blob/master/wp-content/themes/index.php

Reply:
I am unclear, I assumed it was something to signal to wordpress that the directory contained php things, but your reasoning seems more sane.

either way, it existed before, and either I need to back it up, delete around it, or recreate it when I am done. open to either the former or latter, but deleting around it is pretty error prone, if upstream adds more junk in.

Reply:
> Yeah probably. Also is in the official package. Maybe also add it to the themes folder? (Or am I missing something?)

yeah, I did a cute thing, where I create the file once in `wp-content/plugins`, then clone it into `wp-content/themes`, since they are identical.

Reply:
> yeah, I did a cute thing, where I create the file once in `wp-content/plugins`, then clone it into `wp-content/themes`, since they are identical.

I didn't read the bash script too thoroughly, now I see the last line just copies the folder.

Reply:
just let me know if it is too cute, or there is a more succinct/obvious way to do this